soundcard/pulseaudio.py

```diff
@@ -13,8 +13,12 @@
 with open(os.path.join(_package_dir, 'pulseaudio.py.h'), 'rt') as f:
     _ffi.cdef(f.read())
 
-_pa = _ffi.dlopen('pulse')
-
+try:
+    _pa = _ffi.dlopen('pulse')
+except:
+    # Need sometimes explicite sharedobject name on Linux
```

Comment:
Please rephrase: "Try explicit file name, if the general does not work (e.g. on nixos)"